src/main/java/bhs/devilbotz/commands/auto/ScoreAndMobility.java

```diff
@@ -53,7 +53,7 @@ public ScoreAndMobility(Arm arm, DriveTrain drivetrain, double distance, Gripper
             new ArmDown(arm, gripper),
             // Since we are scoring, we always want to drive backwards in the end. Always set
             // negative distance in case the driver forgets
-            new Mobility(drivetrain, -(Math.abs(distance)))));
+            new Mobility(drivetrain, -3)));
```

Comment:
```suggestion
            new Mobility(drivetrain, -(Math.abs(distance)))));
```
same thing, Hard coded to test to see if distance was greyed out because of this, can be put back